src/gsIeti/gsScaledDirichletPrec.hpp

```diff
@@ -0,0 +1,217 @@
+/** @file gsScaledDirichletPrec.hpp
+
+    @brief This class represents the sclaed Dirichlet preconditioner.
+
+    This file is part of the G+Smo library.
+
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+    Author(s): S. Takacs
+*/
+
+#pragma once
+
+#include <gsIeti/gsScaledDirichletPrec.h>
```

Comment:
Please remove the .h file from its hpp counterpart...
Should go inside the _.cpp instead